README.md

```diff
@@ -1,5 +1,7 @@
 # README
 
+Note: See WorkLog.txt in this repo for notes on changes made.
+
```

Comment:
I'm not used to keeping a work log. It may be unnecessary since we can track everything through git history and pull requests.

Reply:
I can leave this out if you like, but it is intended to have more detail that would normally go in a commit message.
It's probably for my use and I can keep a local copy.  Let me know.

Reply:
Are you ok with commenting about the changes here on the PR instead?